### ISSUE TITLE
Correlation of Batchsize and Learning Rate

### ISSUE THREAD

**Issue by ZonghengGuo** [935963004/LaBraM#69]
Thanks for your contribution!

In pretraining stage, if I want to increase batchsize, like from 1024 to 2048, how should I change learning rate?

I guess I need to increase lr to double as well? Thanks for your help in advance.
